Plugins/Flow.Launcher.Plugin.Program/Programs/Win32.cs

```diff
@@ -388,8 +388,7 @@ private static IEnumerable<Win32> UnregisteredPrograms(List<Settings.ProgramSour
                 ShortcutExtension => LnkProgram(x),
                 UrlExtension => UrlProgram(x),
                 _ => Win32Program(x)
-            });
-
+            }).Where(x => x.Valid);
```

Comment:
I think we can do a single check at the end after merging?

Reply:
Sure. That might be better.